### DISCUSSION TITLE
YT just started working again, but the weeks of previous invisible/impossible DLs are still impossible.

### DISCUSSION THREAD

**Discussion by Trelligan** [aclap-dev/video-downloadhelper#1233] [Help and Q&A]
### Your question

   I've been unable to download YT video for weeks. This happens every once in a while, so I did what I normally do: Donated to 'mig' because he has to do more than just the regular upkeep.
      The new donation (without PayPal) unfortunately does not have a 'note' field to tell him what I meant the money for. Bleagh.
      And I've had problems posting here; if you see this I've maybe solved it.

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"3eb05f85e0xxxxxx"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "D:\\Downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "D:\\Downloads"
        }
    },
    "concurrent_downloads_max": 2,
    "show_success_notification": false,
    "never_show_no_incognito_msg_again": true,
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "low_quality": true,
        "sort_by_status": false,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": true,
        "hide_downloaded": true
    },
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 7,
    "successfull_dl": 35,
    "never_show_successfull_dl_message": true,
   [I deleted the Download History for space issues.]

**Comment by Trelligan** [comment]
In order to post this I had to delete my 'download history' as it wouldn't fit in the submission. I also 'xxx'd out the last six digits of my license key for privacy reasons.
   If this wasn't necessary, please let me know.

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

It's good that you worked out how to post here.  It can come in handy when you have a problem to report.  Deleting the history stuff is probably a good idea.  I can imagine cases in which that might come in handy but it's usually not helpful.  On the other hand, for people who think problem resolution happens by ESP or clairvoyance, it can sometimes substitute for a user's failure to post the URL of a video that is causing a problem.  You'd be surprised . . . I'm not any more . . . how often people don't post the URL of something that isn't working properly.

If you couldn't get GitHub to accept your information with all your history, that's rather unusual.  You must have a buttload of history.  Maybe you could delete at least some of that directly from the VDH history page.  I mean, after a certain amount of time, it's not all that useful, is it?

When it comes to your license, I remove mine completely whenever I post my information.  The license number is utterly irrelevant in every way to problem resolution.  It is sometimes relevant to know whether you have a license, but it is never useful to see even a part of the license.  What you did is just fine.

Now to why you are posting.  Why exactly are you posting?  Just to express your appreciation for VDH?  That's always welcome, I'm sure.  Or do you have an actual problem to report?

**Reply by Trelligan** [reply]
I was unable to dl from YT for weeks. As I said, I understand this is a mousetrap problem, YT changes something then VDH changes something.
   The problem is that while I can now DL (at least some) new videos, the backlog of links from those weeks are still not downloading. I also note that, confusingly, my current version of VDH is from 5 Aug, so it hasn't updated yet. 🤔
   1) Notify you guys that videos are now findable/downloadable. ✔️
   2) Let you know that my backlog is still undownloadable. ✔️
   3) Let you guys know that I donate when things are going hard (since there's no longer a note field). ✔️

   Since you asked, here are a couple of URLs I currently can't download, as in VDH doesn't see them while playing. 
https://www.youtube.com/watch?v=rjDTWndH_bc
https://www.youtube.com/watch?v=E38y5jREr3U

**Reply by GrampaWildWilly** [reply]
Let me repeat.  I do not work for VDH.  I'm just another user.

My setup is Windows 11 Pro 64-bit version 23H2 build 22631.4112, Firefox 130.0 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

With that environment, VDH downloaded both those items without a problem, the first one in just a few seconds, the second in just under a minute & a half.  I didn't even have to launch playback first.  They are not like most videos.  The first looks like a series of comic book pages.  The second is just a single image that also looks like it might be from a comic book.  Both have sound tracks of a person narrating the story, accompanied by visible subtitles.  I don't know why you wouldn't be able to download those.  They download fine for me.

I note that you are using the latest -- yes, it is the latest, regardless of the date -- general availability version.  But since you are running Firefox, you really ought to get the beta.  Maybe it will help with this problem.

**Comment by Trelligan** [comment]
Thanks, downloading Beta.

    On Monday, September 9, 2024 at 09:36:45 AM PDT, GrampaWildWilly ***@***.***> wrote:  
 
 


Let me repeat. I do not work for VDH. I'm just another user.

My setup is Windows 11 Pro 64-bit version 23H2 build 22631.4112, Firefox 130.0 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

With that environment, VDH downloaded both those items without a problem, the first one in just a few seconds, the second in just under a minute & a half. I didn't even have to launch playback first. They are not like most videos. The first looks like a series of comic book pages. The second is just a single image that also looks like it might be from a comic book. Both have sound tracks of a person narrating the story, accompanied by visible subtitles. I don't know why you wouldn't be able to download those. They download fine for me.

I note that you are using the latest -- yes, it is the latest, regardless of the date -- general availability version. But since you are running Firefox, you really ought to get the beta. Maybe it will help with this problem.
